docs/setup/repo-and-action.md

```diff
@@ -28,29 +28,13 @@ Please follow the steps below to configure the connection information of TiDB cl
    mysql://<user>:<password>@<host>:<port>
```

Comment:
@634750802  Can the DATABASE_URL part be designed to be compatible with dbname, for instance, the connect string in serverless usually includes dbname, defaulting to 'test' or 'defaultdb' and so on. This covers most use cases where users need to connect to a specific dbname. After a user copies it, the dbname can be used directly. Therefore, connect strings that users copied will contain the dbname. I wonder if it's possible to add a logic to Lite, where if a user inputs a dbname, it will automatically get ignored.